content/en/docs/reference/command-line-tools-reference/feature-gates/api-serving-with-routine.md

```diff
@@ -0,0 +1,14 @@
+---
+title: APIServingWithRoutine
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
```

Comment:
this is reverted to alpha / disabled by default just this week, due to regressions it caused

https://github.com/kubernetes/kubernetes/pull/126470

Reply:
OK, but:
- _this_ PR should mark it as beta
- a separate PR needs to record the demotion to alpha for v1.31 (please target _dev-1.31_)

Reply:
it's getting demoted to alpha and disabled by default in 1.30 as well because it regressed - https://github.com/kubernetes/kubernetes/pull/126481

Reply:
OK, we'd like a text note about the specific patch releases where it was beta and enabled by default. That detail is useful to users.